cmd/ci-operator/main.go

```diff
@@ -286,7 +286,9 @@ func (o *options) Complete() error {
 		if err != nil {
 			return fmt.Errorf("failed to determine job spec: failed to resolve --git-ref: %v", err)
 		}
+		pulls := jobSpec.Refs.Pulls
```

Comment:
This is going to be surprising, no? In the rehearsal case we're going to see a Refs from --git-ref for target repo, but a Refs.Pulls from Prow for the release PR?

Reply:
Exactly.